README.md

```diff
@@ -1,8 +1,11 @@
-<img src="https://s3.amazonaws.com/aws-mobile-hub-images/aws-amplify-logo.png" alt="AWS Amplify" width="225">
+<img src="./docs/public/amplify-logo.svg" alt="AWS Amplify Logo" width="25"> AWS Amplify
 
 ---
 
-# Amplify UI
+<p align="center">
+  <img src="docs/public/svg/favicon.svg" width="150vw" alt="Amplify UI logo" />
```

Comment:
I don't think we want `150vw` for the width here? The `vw` unit is based on the width of the browser and this would be 150% of the browser's width. Also the `width` attribute on an image is a unitless attribute, this would just mean "150px".

Reply:
Good catch. Updated.